history.txt

```diff
@@ -1,116 +1,77 @@
-- added ability to transfer all the data of a given user using CLI
-- added support for PyTables version 3.4+
-- deprecated Path Object in OMERO model
-- updated URL to point to new website, docs and downloads
-- added method to OMERO.py to manipulate advanced rendering settings
-- installed django-redis when installing web requirements
-- added tests for cleanse
-- reviewed calculation of stats in RoiService
-
-
-This release also upgrades the version of Bio-Formats which OMERO uses to 5.7.1
-
-
-5.4.0-m3 (August 2017)
-----------------------
-
-Developer preview release. Intended *only as a developer preview* for updating
-code before the full public release of 5.4.0. **Use at your own risk.**
-
-Changes include:
-
-- improved display of plate in OMERO.web
-- various improvements in integration tests
-- fixed import related bugs in OMERO.insight
-- fixed synchronization of controls enabling copy/paste of
-  rendering settings in OMERO.web
-- fixed projection of images
-- removed Scipy dependency
-
-This release also upgrades the version of Bio-Formats which OMERO uses to 5.6.0-m1
-
-5.4.0-m2 (July 2017)
+5.4.0 (October 2017)
 --------------------
 
-Developer preview release. Intended *only as a developer preview* for updating
-code before the full public release of 5.4.0. **Use at your own risk.**
-
-Changes include:
-
-- fixed OMERO.server permissions bug with restricted administrators
-- adjusted OMERO.blitz API to allow some query results to be cached
-- improve OMERO.web experience and fixed bugs for restricted administrators
-- improve OMERO.cli:
-  - fixed admin plugin so "cleanse" can handle larger directories
-  - have chgrp plugin check administrator Chgrp restriction
-  - added to chown plugin ability to target all of given users' data
-  - adjusted handling of standard input
-- various improvements in integration tests
+A full, production-ready release of OMERO 5.4.0; featuring a new configurable
+user role "Restricted Administrators"; further improvements to OMERO.web;
+additions to OMERO.cli; and many fixes and performance improvements:
+
+-  added :doc:`/sysadmins/restricted-admins` to allow sysadmins to delegate
+   management tasks to facility managers without granting them full system
+   admin privileges, or to allow trusted users such as image analysts to carry
+   out tasks on behalf of all other users
+-  fixed color conversion to RGBA
+-  added support for exporting images in a plate as OME-TIFF
+-  improved creation of rendering settings for image without any stats
+   e.g. 32bit images
+-  improved performance for moving large plates
+-  fixed import bug in OMERO.insight
+-  fixed projection of images if the range of timepoints specified is not
+   the full range
+-  added support for transfering ownership of all the data of a given user
+   using CLI
+-  renamed "Reverse Intensity" command to "Invert" in image viewers
+-  fixed label position for wells
+-  OMERO.web UI fixes:
```

Comment:
Add a line before to make it easier to read?

Reply:
The line between this and the subpoints isn't rendered in the final content, I had to add it to get the formatting correct because the CSS is doing something funny and was making the preceding points doublespaced.

Reply:
thanks for the explanation.
Ignore in that case